spec/requests/sitemaps_spec.rb

```diff
@@ -40,5 +40,39 @@
       expect(response.body).not_to include(articles.last.path)
       expect(response.media_type).to eq("application/xml")
     end
+
+    it "renders most recent posts if /sitemap-posts" do
```

Comment:
_**Quick, non-blocking suggestion:**_
```suggestion
    it "renders most recent posts if /sitemap-posts", :aggregate_failures do
```

If you decide to move forward with aggregating the failures here, then I think we should also do the same for the other specs that have multiple expectations!